libfuse/dir.go

```diff
@@ -347,6 +353,12 @@ func (d *Dir) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.Lo
 	d.folder.fs.log.CDebugf(ctx, "Dir Lookup %s", req.Name)
 	defer func() { d.folder.reportErr(ctx, libkbfs.ReadMode, err) }()
 
+	err = libkbfs.EnterCriticalWithTimeout(
```

Comment:
Yes possibly, plus in tests the goroutines/memory from previous tests could stick around a long time, making it more difficult and confusing to debug stacktraces.